### DISCUSSION TITLE
0.6.0-beta 3

### DISCUSSION THREAD

**Discussion by mdelobelle** [mdelobelle/metadatamenu#422] [Announcements]
Reminder for new features in this beta: https://github.com/mdelobelle/metadatamenu/discussions/420#discussion-5813183

since 0.6.0-beta 2
improved:
- manage lookup field manual change (manually changed fields are skipped in update lookups so as not to be overwritten

fixed:
- sorting for nested fields in settings and fileclass fields
- fixed lookup not added through commands
- fixed lookups and formulas not resolving properly
- arrayAsStringIsEqual fixed when values are arrays
- removed previousFileLookupFilesValues not needed anymore
- remove stale items from indexedDB after file rename or file delete

<hr /><em>This discussion was created from the release <a href='https://github.com/mdelobelle/metadatamenu/releases/tag/0.6.0-beta-3'>0.6.0-beta 3</a>.</em>

**Comment by arminta7** [comment]
I've been getting this popup occasionally:

<img width="319" alt="Screenshot 2023-11-06 at 12 52 59" src="https://github.com/mdelobelle/metadatamenu/assets/7927356/00d02296-4f2f-45ca-a586-e82bec60f8eb">

**Reply by mdelobelle** [reply]
I can't reproduce with your fileClass definitions. Do you have any error in the console when this is occuring?

**Reply by arminta7** [reply]
I don't have any error. This is only in my bigger vault copy, not my fresh tester vault. Could it be related to migration? It looks they are all formulas. Idk.
